node/bft/src/helpers/storage.rs

```diff
@@ -774,18 +774,18 @@ impl<N: Network> Storage<N> {
         block: &Block<N>,
         certificate: BatchCertificate<N>,
         unconfirmed_transactions: &HashMap<N::TransactionID, Transaction<N>>,
-    ) {
+    ) -> Result<()> {
```

Comment:
Given that this function only returns Ok(()), I guess this is a useless change and can be reverted... Just trying to keep the scope of the PR small and avoid potential oversights.